estuary_updater/handlers/distgit.py

```diff
@@ -23,11 +23,28 @@ def can_handle(msg):
         :return: a bool based on if the handler can handle this kind of message
         :rtype: bool
         """
-        return msg['topic'] == '/topic/VirtualTopic.eng.distgit.commit'
+        supported_topics = [
+            '/topic/VirtualTopic.eng.distgit.commit',
+            '/topic/VirtualTopic.eng.distgit.push'
+        ]
+        return msg['topic'] in supported_topics
 
     def handle(self, msg):
         """
-        Handle a dist-git message and update Neo4j if necessary.
+        Handle a dist-git message by sending it to the right handler method and update Neo4j.
+
+        :param dict msg: a message to be processed
+        """
+        if msg['topic'] == '/topic/VirtualTopic.eng.distgit.commit':
+            self.distgit_commit_handler(msg)
+        elif msg['topic'] == '/topic/VirtualTopic.eng.distgit.push':
+            self.distgit_push_handler(msg)
+        else:
+            raise RuntimeError('This message is unable to be handled: {0}'.format(msg))
+
+    def distgit_commit_handler(self, msg):
```

Comment:
Optional: Since the class is called `DistGitHandler`, prefixing the methods with `distgit_` is somewhat redundant. The prefix could be removed.